src/server/SocketServer/index.js

```diff
@@ -0,0 +1,49 @@
+const socketIo = require('socket.io');
+
+/**
+ * Abstract Api of socket.io lib
+ */
+class SocketServer {
+  /**
+   * Create a websocket whith an instant of http server
+   * @param {httpServer} httpServer
```

Comment:
Why to add comments here ? And nowhere else ?

What are your thoughts on comments by the way ? 🤔

Reply:
I thought I'd put it on for everyone. But I haven't done it yet. Is it better or is it not necessary.